templates/ts-apollo-fullstack/server/src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { loadConfigSync } from 'graphql-config';
+import Knex from 'knex';
+
+export function getProjectConfig() {
```

Comment:
I talked about that with our community. We really need that stuff exposed in the CLI so we can avoid asking people to have this util copied into their code. This is very generic and it should not change at all.

Reply:
Agreed.

Reply:
The only issue is graphql-cli feature to recognize projects. If we load that directly we might not support project use cases, but we do not support that here anyway.

Reply:
Ignore previous convoluted comments. I think your proposal makes sense.